packages/testlab/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A collection of test utilities we use to write LoopBack tests.",
   "main": "index",
   "scripts": {
-    "build": "tsc && npm run build:es2015",
-    "build:es2015": "tsc --target es2015 --outDir lib-es2015",
+    "build": "npm run build:es2015 && npm run build:es2017",
+    "build:es2015": "tsc --target es2015 --outDir lib6",
+    "build:es2017": "tsc --target es2017 --outDir lib",
```

Comment:
@bajtos This should address https://github.com/strongloop/loopback-next/pull/225#discussion_r113669953.

Annoyingly, the `--outDir` must be `'es3', 'es5', 'es6', 'es2015', 'es2016', 'es2017', 'esnext'` (so we can't have `es7` for example).